server/venueless/live/modules/chat.py

```diff
@@ -440,6 +441,35 @@ async def send(self, body):
                     }
                 )
 
+    @command("react")
+    @channel_action(
+        room_permission_required=Permission.ROOM_CHAT_SEND,
+        room_module_required="chat.native",
+        require_membership=True,
+    )
+    async def react(self, body):
+        reaction = body["reaction"]
+        if reaction not in emoji.UNICODE_EMOJI:
+            raise ConsumerException("emoji.sadpanda")
```

Comment:
Have we checked that backend and frontend using… uhm… similar versions of Unicode to determine which emoji are available?

Reply:
We haven't, and honestly, I could just as well disable the check and trust the frontend.

Reply:
well, I'd suggest to leave it for now and see if it breaks. not validating at all sounds like it may lead to nasty situations as well